docs/contributing/998-archived/01-compile-targets.md

```diff
@@ -91,7 +91,7 @@ In this example, we want to sleep briefly for the Simulator target and for 30 se
 
 Compiler plugins serve to extend the capabilities of the Wing compiler. They offer customization for the compilation output, such as infrastructure definitions. For instance, a compiler plugin can be employed to enforce a specific tagging convention on all resources within the resulting infrastructure.
 
-Learn more about compiler plugins [here](https://www.winglang.io/docs/tools/compiler-plugins)
+Learn more about compiler plugins [here](https://www.winglang.io/contributing/archived/compiler-plugins)
```

Comment:
```suggestion
Learn more about platforms [here](https://www.winglang.io/docs/concepts/platforms)
```

Reply:
Same remark as above. This is an archived doc.